encrypted-media.js

```diff
@@ -408,8 +408,6 @@
           var prefix = is_registry_file ? "../../" : "";
           EME_spec_url = prefix + file;
           groupBaseURLs[x] = EME_spec_url;
-          // Refer to the Web IDL Editor’s Draft from Editor’s Drafts of this spec.
-          IDL_spec_url = "https://webidl.spec.whatwg.org/";
```

Comment:
@tidoust I updated all the spec URL's, so I think this is unnecessary. Do you agree?

Reply:
Not sure what you mean by "unnecessary"? The goal of the update was to avoid a redirect and use the right URL for Web IDL directly, which we try to do in all specs.

Reply:
@tidoust **IDL_spec_url** is already defined on line 5 of encrypted-media.js and I updated that to point to [https://webidl.spec.whatwg.org/](https://webidl.spec.whatwg.org/), so lines 411-412 are unnecessary.

Reply:
Ok, sounds good then!